whitepaper/chapter01.md

```diff
@@ -143,4 +143,4 @@ As a result, telcos will be able to freely test, design, deploy, provision, revi
 
 Work is ongoing to bring the most demanding and mission critical telecommunications systems to cloud native, including those that must fulfil regulatory requirements (such as emergency call handling). However, in theory, CNFs running on Kubernetes should provide equivalent regulatory compliance for the same reasons that cloud native applications for enterprises tend to be more fault-tolerant, more responsive and more resilient; because loosely coupled immutable systems are inherently better at handling highly complex applications with strict SLAs and multiple dependent and interactive functions. This is the significant promise of cloud native for telecommunications applications and functions.
 
-Contributors: [Bill Mulligan](https://github.com/xmulligan),[Dan Kohn](https://github.com/dankohn),[Daniel Bernier](https://github.com/dabernie),[Dave Cremins](https://github.com/davecremins),Ervins Kampans,[Frederick Kautz](https://github.com/fkautz),[Gergely Castari](https://github.com/CsatariGergely),Henrik Saveedra Persson,[Herbert Damker](https://github.com/hdamker),[Ildiko Vancsa](https://github.com/ildikov),[Jeff Saelens](https://github.com/jeffsaelens),Khawaja Daniyal,Lei Wang,Saad Sheikh,[Tamas Zsiros](https://github.com/TamasZsiros),[Taylor Carpenter](https://github.com/taylor),[Tom Kivlin](https://github.com/tomkivlin),[W. Watson](https://github.com/wavell)
+Contributors: [Bill Mulligan](https://github.com/xmulligan),[Dan Kohn](https://github.com/dankohn),[Daniel Bernier](https://github.com/dabernie),[Dave Cremins](https://github.com/davecremins),Ervins Kampans,[Frederick Kautz](https://github.com/fkautz),[Gergely Castari](https://github.com/CsatariGergely),Henrik Saveedra Persson,[Herbert Damker](https://github.com/hdamker),[Ildiko Vancsa](https://github.com/ildikov),[Jeff Saelens](https://github.com/jeffsaelens),Khawaja Daniyal,Lei Wang,Saad Sheikh,[Tamas Zsiros](https://github.com/TamasZsiros),[Taylor Carpenter](https://github.com/taylor),[Tom Kivlin](https://github.com/tomkivlin),[W. Watson](https://github.com/wavell), [Ahmed ElSawaf] (https://github.com/ASawwaf)
```

Comment:
You need to remove the space between the `]` and the `(`.  Otherwise, no problem with this change.